README.md

```diff
@@ -57,7 +57,7 @@ first time to run indexer you need to create the database schema
 ```shell
 $ mkdir -p ~/indexer-data/mysql ~/indexer-data/geth
 # Create database sechema
-MYSQL_DATA_PATH="$HOME/indexer-data/mysql" docker-compose up ws-database ws-migration
+MYSQL_DATA_PATH="$HOME/indexer-data/mysql" docker-compose up idx-database idx-migration
 # press Ctrl + C when see `eth-indexer_ws-migration_1 exited with code 0`
```

Comment:
the name also need to be rename to `eth-indexer_idx-migration_1`

Reply:
Because the command is `docker-compose`. The name should be consistent with `docker-compose.yml`, not the name of the running container.

Reply:
Oh, it's my misunderstanding. Fixed.